test/user.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+const chai = require('chai')
+const expect = chai.expect
+const nock = require('nock')
+const mojang = require('../')
+
+describe('mojang.user()', () => {
```

Comment:
Maybe you should setup [`after` or `afterEach`](http://stackoverflow.com/questions/16747367/mocha-js-to-run-after-hook-even-if-test-suit-fails) hooks here (and/or inside `username` tests), that does [`nock.cleanAll()`](https://github.com/node-nock/nock#cleanall)